meilisearch/client.py

```diff
@@ -35,10 +35,9 @@ def create_index(self, uid, primary_key=None, name=None):
         ----------
         uid: str
             UID of the index
-        primary_key: str, optional
-            Attribute used as unique document identifier
-        name: str, optional
-            Name of the index
+        options: dict, optional
+            Options passed during creation (ex: primaryKey)
```

Comment:
```suggestion
            Options passed during index creation (ex: primaryKey)
```
To avoid confusion about which creation